Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ZoomableLineEdit.h

```diff
@@ -0,0 +1,37 @@
+
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <QLineEdit>
+
+namespace EMStudio
+{
+
+    class ZoomableLineEdit : public QLineEdit
```

Comment:
update: An API comment here would be very helpful to describe the context/usage scenario

Reply:
Comment is great thanks